### PR TITLE
Addition of PGGD collection

### PR DESCRIPTION
**Pull request by KellyVance**
**Changes**

Addition of collections
pggd-01: drilling-of-borehole-on-notice ;
pggd-02: completion, bhse:alteration, bhse:abandonment ;
pggd-03: hydraulic-fracturing-on-notice;
pggd-04: bhse:hydraulic-fracturing-abandoned, bhse:hydraulic-fracturing-completed;

**Reviewers**
SMEs @LizDerrington @BlakePaul Is the above correct? Not sure if drilling-abandoned or fraccing-abandoned belong in with the PGGD forms at all. Please let me know.
Tech @LukeHauck and @DavidCrosswellGSQ 

FYI - @lisa-woody

### CODE REVIEW

**Comment by LukeHauck** [comment]
Undefined prefix "gsqdt:" on line 130.

**Comment by KellyVance** [comment]
> Undefined prefix "gsqdt:" on line 130.

fixed! thanks.

File: vocabularies/borehole-status-event.ttl [hunk 1]
```diff
@@ -11,19 +11,20 @@
     dcterms:creator <http://linked.data.gov.au/org/gsq> ;
     dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
     dcterms:created "2020-01-21"^^xsd:date ;
-    dcterms:modified "2020-06-17"^^xsd:date ;
+    dcterms:modified "2020-07-07"^^xsd:date ;
     dcterms:source "Compiled by the Geological Survey of Queensland" ;
     skos:definition "Events that occur during the lifecycle a borehole that affect the status or history, or convert a borehole from one purpose to another."@en ;
     skos:hasTopConcept bhse:drilling-of-borehole-on-notice,
+        bhse:drilling-of-borehole-abandoned,
```
Comment (DavidCrosswellGSQ) [inline vocabularies/borehole-status-event.ttl:18]: Should this be "Never drilled" as to clearly delineate it from bhse:abandonment?

File: vocabularies/borehole-status-event.ttl [hunk 2]
```diff
@@ -123,3 +124,28 @@ bhse:manual-update a skos:Concept ;
...
+    skos:prefLabel "PGGD-02"@en .
+
+bhse:pggd-03 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:hydraulic-fracturing-on-notice;
+    skos:prefLabel "PGGD-03"@en .
+
+bhse:pggd-04 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:hydraulic-fracturing-abandoned,
```
Comment (DavidCrosswellGSQ) [inline vocabularies/borehole-status-event.ttl:149]: @BlakePaul @LizDerrington 
The other possible states for the current PGGD-04 form are Partial Completion, Alteration and Other?
I think we need to add Partial Completion and Alteration.
Can we get rid of Other? Or replace it with something more meaningful?

File: vocabularies/borehole-status-event.ttl [hunk 3]
```diff
@@ -123,3 +124,28 @@ bhse:manual-update a skos:Concept ;
     skos:inScheme <http://linked.data.gov.au/def/borehole-status-event> ;
     skos:prefLabel "Manual Update or Creation"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
+
+#COLLECTIONS
+
+bhse:pggd-01 a skos:Collection ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl]: removed

File: vocabularies/borehole-status-event.ttl [hunk 4]
```diff
@@ -123,3 +124,28 @@ bhse:manual-update a skos:Concept ;
...
+    skos:prefLabel "PGGD-01"@en .
+
+bhse:pggd-02 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:completion,
+        bhse:alteration,
+        bhse:abandonment ;
+    skos:prefLabel "PGGD-02"@en .
+
+bhse:pggd-03 a skos:Collection ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl]: removed

File: vocabularies/borehole-status-event.ttl [hunk 5]
```diff
@@ -123,3 +124,28 @@ bhse:manual-update a skos:Concept ;
...
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
+    skos:member bhse:hydraulic-fracturing-on-notice;
+    skos:prefLabel "PGGD-03"@en .
+
+bhse:pggd-04 a skos:Collection ;
+    skos:definition "The borehole events that may be reported via a PGGD-02 form to change the status of a borehole."@en ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl]: done

File: vocabularies/borehole-status-event.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl]: changed

File: vocabularies/borehole-status-event.ttl (hunk 2 above)
Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl:163]: partial and altered added
other added as Hydraulic fracturing related events

File: vocabularies/borehole-status-event.ttl [hunk 6]
```diff
@@ -83,6 +87,24 @@ bhse:hydraulic-fracturing-completed a skos:Concept ;
...
+    skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
+
+bhse:hydraulic-fracturing-partial a skos:Concept ;
+    skos:definition "A Notice of a partial completion of hydraulic fracturing activities. The full well fracturing objectives were not acheived, but a portion of the planned hydraulic fracturing activities were carried out. Lodgement of Form PGGD-04. This event is a recording of a reportable activity which does not change the status"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/borehole-status-event> ;
+    skos:prefLabel "Hydraulic fracturing partially completed"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
+
+bhse:hydraulic-fracturing-related a skos:Concept ;
+    skos:definition "A Notice of a other events relating to hydraulic fracturing activities. Any other related to hydraulic fracturing such as: industrial accident/s, adverse evironmental condition/s (such as too wet, too hot, electrical storm etc.) at the wellsite, or concerns over workplace health and safety. Lodgement of Form PGGD-04. This event is a recording of a reportable activity which does not change the status"@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/borehole-status-event.ttl:103]: As you've used special characters, you will need to wrap this paragraph in 3 quotation marks.

Comment (DavidCrosswellGSQ) [inline vocabularies/borehole-status-event.ttl:103]: Remove the "a" from _A Notice of a other events_

File: vocabularies/borehole-status-event.ttl [hunk 7]
```diff
@@ -83,6 +87,24 @@ bhse:hydraulic-fracturing-completed a skos:Concept ;
     skos:prefLabel "Hydraulic fracturing completed"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
 
+bhse:hydraulic-fracturing-altered a skos:Concept ;
+    skos:definition "A Notice of completion of hydraulic fracturing activities but wth alteration. Lodgement of Form PGGD-04. The full well fracturing objectives were required to be changed due to unforeseen circumstances throughout the process, but hydraulic fracturing activities were carried out. This event is a recording of a reportable activity which does not change the status"@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/borehole-status-event.ttl:91]: Correct _wth_

File: vocabularies/borehole-status-event.ttl (hunk 6 above)
Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl]: done

Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl]: done

File: vocabularies/borehole-status-event.ttl (hunk 7 above)
Comment (KellyVance) [reply vocabularies/borehole-status-event.ttl]: done

**Pull request merged**